visualization/dashboard/src/Localization/en.json

```diff
@@ -186,5 +193,150 @@
+    },
+    "Filters": {
+        "equalComparison": "Equal to",
+        "greaterThanComparison": "Greater than",
+        "lessThanComparison": "Less than",
+        "inTheRangeOf": "In the range of",
+        "categoricalIncludeValues": "Included values:",
+        "numericValue": "Value",
+        "numericalComparison": "Operation",
+        "minimum":"Minimum",
+        "maximum":"Maximum"
+    },
+    "Columns": {
+        "regressionError": "Regression error",
+        "_regressionError.comment": "true value minus predicted value is regression error",
+        "error": "Error",
+        "classificationOutcome": "Classification outcome",
+        "_classificationOutcome.comment": "Whether the prediction matched the truth for classification, true positive, true negative, false positive, false negative are the values",
+        "truePositive": "True positive",
+        "trueNegative": "True negative",
+        "falsePositive": "False positive",
+        "falseNegative": "False negative",
+        "dataset": "Dataset"
+    },
+    "WhatIf": {
+        "closeAriaLabel": "Close",
+        "panelHeader": "What if Panel",
+        "indexErrorMessage": "Error: index must be an integer between 0 and {0}",
+        "noCustomPoints": "No custom points",
+        "defaultCustomRootName": "Copy of row {0}",
+        "colorLabel": "Select color:",
+        "filterFeaturePlaceholder": "Search features"
+    },
+    "Cohort": {
+        "cohort": "Cohort",
+        "defaultLabel": "All data"
+    },
+    "GlobalTab": {
+        "secondaryChart": "Select chart type:",
+        "helperText": "Explore the top k important features that impact your overall model predictions. Use the slider to show descending feature importances . Select up to three cohorts to see their feature importances side by side. Click on any of the features in the graph to see a density plot below of how values of the selected feature affect prediction.",
+        "topAtoB": "Top {0}-{1} features",
+        "datasetCohorts": "Dataset cohorts",
+        "legendHelpText": "Toggle cohorts on and off in the plot by clicking on the legend items.",
+        "sortBy": "Sort by",
+        "viewDependencePlotFor": "View dependence plot for:",
+        "datasetCohortSelector": "Select a dataset cohort",
+        "aggregateFeatureImportance": "Aggregate Feature Importance"
+    },
+    "FeatureBar": {
+        "sortBy": "Sort by:"
+    },
+    "CohortBanner": {
+        "dataStatistics": "Data Statistics",
+        "datapoints": "{0} datapoints",
+        "features": "{0} features",
+        "filters": "{0} filters",
+        "binaryClassifier": "Binary Classifier",
+        "regressor": "Regressor",
+        "multiclassClassifier": "Multiclass Classifier",
+        "datasetCohorts": "Dataset Cohorts",
+        "editCohort": "Edit Cohort",
+        "duplicateCohort": "Duplicate Cohort",
+        "addCohort": "Add Cohort",
+        "copy": " copy"
```

Comment:
is the space in " copy" needed?

Reply:
yes, its a suffix following the name of a cohort being copied. Can be written as a formatted string like '{0} copy'